wiki/Guides/en.md

```diff
@@ -49,3 +47,10 @@ Below is a sorted list of guides created by members of the osu!community; most o
 - [Cropping with Complex Backgrounds](Cropping_with_Complex_Backgrounds)
 - [Cropping with Simple Backgrounds](Cropping_with_Simple_Backgrounds)
 - [Making Properly Centered Spinners](Making_Properly_Centered_Spinners)
+
+## Miscellaneous
+
+- [Discord RPC](Discord_RPC)
```

Comment:
this is wrong. it should be "Rich Presence".

it's not RPC. that's a developer term which shouldn't